cql2pgjson/src/main/java/org/folio/cql2pgjson/util/Cql2SqlUtil.java

```diff
@@ -140,10 +137,7 @@ public static String cql2string(String s) throws QueryValidationException {
   @SuppressWarnings("squid:S3776")  // suppress "Cognitive Complexity of methods should not be too high"
   public static String cql2regexp(String s) {
     StringBuilder regexp = new StringBuilder();
-    /**
-     * true if the previous character is an escaping backslash
-     */
-    boolean backslash = false;
+    boolean backslash = false;  // previous character is backslash
```

Comment:
Can you re-insert "escaping" before backslash?
Can you re-insert "true if" at the begin of the comment to show that the comment is not about "false".